phial/wrappers.py

```diff
@@ -68,6 +68,84 @@ def __eq__(self, other: object) -> bool:
         return self.__dict__ == other.__dict__
 
 
+class MessageAttachmentField():
+    '''
+    Displays field in message attachment
+
+    Attributes:
+        title(str): The title of the field
+        value(str): The value of the field
+        short(bool): If false, 2 fields will be displayed on the same line
+
+    '''
+    def __init__(self,
+                 title: Optional[str] = None,
+                 value: Optional[str] = None,
+                 short: Optional[bool] = None) -> None:
+        self.title = title
+        self.value = value
+        self.short = short
+
+    def __repr__(self) -> str:
+        return "<Message Attachment Field with title {0} >".format(self.title)
+
+
+class MessageAttachment():
+    '''
+    Displays a message attachment
+
+    Attributes:
+        fallback(str): Plain text summary of the attachment
+        color(str): Hex code beginning with a '#' for the side bar colour
+        author_name(str): Small text to display the authors name
+        author_link(str): URL for link on authors name
+        author_icon(str): URL for 16x16 icon beside the authors name
+        title(str): The title of the attachment
+        title_link(str): A URL for the attachment title
+        text(str): The main text body of an attachment
+        image_url(str): URL to an image file that will be displayed with
+                        an attachment
+        thumb_url(str): URL to an image file that will be displayed as a
+                        thumbnail on the right side of a message attachment
+        fields(list): A list of MessageAttachmentFields to display on the
+                      attachment
+        footer(str): Small text to help contextualize and identify an
+                     attachment
+        footer_icon(str): URL for 16x16 icon beside the footer
+
+    '''
+    def __init__(self,
+                 fallback: Optional[str] = None,
```

Comment:
Same as the comment above in `MessageAttachmentField`. A brief glance at the docs seems to say that some of these are optional, which implies some are not. It would be great to make Slack required fields, required in our models.

Reply:
Again as with the attachment field, an empty object is accepted just isn't displayed in any way